igvm/templates/domain.xml

```diff
@@ -1,12 +1,12 @@
 <domain type='kvm'>
-  <name>{{ fqdn }}</name>
+  <name>{{ name }}</name>
   <uuid>{{ props.uuid }}</uuid>
```

Comment:
While we're at this.. why not use Serveradmin object ID as UUID for a VM?

Reply:
> The content of the uuid element provides a globally unique identifier for the virtual machine. The format must be RFC 4122 compliant, eg 3e3fce45-4f53-4fa7-bb32-11f34168b82b.

Reply:
Object_id is not a valid uuid